contribs/gmf/src/directives/mobilemeasurepoint.js

```diff
@@ -0,0 +1,202 @@
+goog.provide('gmf.MobileMeasurePointController');
+goog.provide('gmf.mobileMeasurePointDirective');
+
+goog.require('gmf');
+goog.require('gmf.Altitude');
+goog.require('ngeo.Debounce');
+goog.require('ngeo.interaction.MeasurePointMobile');
+goog.require('ngeo.interaction.MobileDraw');
+
+gmfModule.value('gmfMobileMeasurePointTemplateUrl',
+    /**
+     * @param {angular.JQLite} element Element.
+     * @param {angular.Attributes} attrs Attributes.
+     */
+    function(element, attrs) {
+      var templateUrl = attrs['gmfMobileMeasurePointTemplateurl'];
+      return templateUrl !== undefined ? templateUrl :
+          gmf.baseTemplateUrl + '/mobilemeasurepoint.html';
+    });
+
+
+/**
+ * Provide a "mobile measure" directive.
+ *
+ * @example
+ * <div gmf-mobile-measure-point=""
+ *   gmf-mobile-measure-point-map="::ctrl.map"
+ *   gmf-mobile-measure-point-measure-interaction="::ctrl.measure"
+ *   gmf-mobile-measure-point-draw-interaction="ctrl.drawInteraction">
+ * </div>
+ *
+ * @param {string|function(!angular.JQLite=, !angular.Attributes=)}
+ *     gmfMobileMeasurePointTemplateUrl Template URL for the directive.
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname gmfMobileMeasurePoint
+ */
+gmf.mobileMeasurePointDirective =
+    function(gmfMobileMeasurePointTemplateUrl) {
+  return {
+    restrict: 'A',
+    scope: {
+      'map': '=gmfMobileMeasurePointMap',
+      'measure': '=gmfMobileMeasurePointMeasureInteraction',
```

Comment:
The map is needed for the altitude service.
The measure is used to be listened to on 'active' change.